lib/style-transform.js

```diff
@@ -184,6 +184,7 @@
 
                             // animation name is anything not in this list
                             if (
+                              prop &&
```

Comment:
nvm ignore this comment, I overlooked this I thought that `isNaN(parseInt(prop))` was negated `! ...`.